server/shared/store/index.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const autobind = require('auto-bind');
+const config = require('../../../config/server').store;
+const path = require('path');
+
+class Store {
+  constructor(config) {
+    this.provider = Store.createProvider(config);
+    autobind(this);
+  }
+
+  /**
+   * @param {number} ttl - time to live measured in seconds
+   */
+  set(key, value, ttl) {
+    return this.provider.set(key, value, ttl);
+  }
+
+  get(key) {
+    return this.provider.get(key);
+  }
+
+  has(key) {
+    return this.provider.has(key);
+  }
+
+  keys(pattern = '*') {
```

Comment:
Consider renaming this method to `getKeys` to indicate that it's a function, not a class field.